twine-upload.sh

```diff
@@ -0,0 +1,7 @@
+#! /usr/bin/env bash
+set -Eeuo pipefail
+
+TWINE_USERNAME="$INPUT_USER" \
+TWINE_PASSWORD="$INPUT_PASSWORD" \
+TWINE_REPOSITORY_URL="$INPUT_REPOSITORY_URL" \
```

Comment:
@mfussenegger have you checked that it works with an empty default for `TWINE_REPOSITORY_URL`?

Reply:
Yep I think I did.

Reply:
Thanks for the confirmation. I haven't checked yet because I hit a bug with `deployment` event: https://twitter.com/mscccc/status/1166802969863438337